orga/translations/en.json

```diff
@@ -31,6 +32,10 @@
     }
   },
   "pages": {
+    "certifications": {
+      "description": "Jiberrish",
```

Comment:
"You can export the certification results for all divisions in CSV format." ?

Reply:
On attendait la version "officielle" censée être discutée demain matin. Cf discussion Slack sur traduction-app.

Reply:
@iwazaru Les fichiers de trad sont faits par une orga externe :)

Reply:
Dully noted!

Reply:
Correction : la grosse partie de la traduction de pixapp était en effet réalisée par une orga externe, aujourd'hui ce sont des internes qui ont pris la suite. Il faut les solliciter via le channel traduction-app